main.cpp

```diff
@@ -35,51 +35,51 @@ Sub Object 1: Steering Wheel
     Name 4 things you'll find on the:    Steering Wheel
         1) paddle shifters
         2) 'cruise control' controls
-        3) 
-        4) 
+        3) horn
+        4) airbag
     Name 2 things you can do with the:   Steering Wheel
         1) adjust cruise control settings.
-        2)
+        2) press horn
         
 Sub Object 2: Instrument Cluster
     Name 4 things you'll find on the:   Instrument Cluster
-        1)
-        2)
-        3)
-        4)
+        1) speedometer
+        2) rpm meter
+        3) computer display
+        4) lights indicator
     Name 3 things you can do with the:   Instrument Cluster
-        1)
-        2)
-        3)
+        1) monitor speed
+        2) monitor rpm
+        3) monitor lights
     
 Sub Object 3: Environment Controls
     Name 3 things you'll find on the:    Environment Controls
-        1)
-        2)
-        3)
+        1) air Conditioner controls
+        2) radio controls
+        3) watch
     Name 3 things you can do with the:   Environment Controls
-        1)
-        2)
-        3)
+        1) adjust environment temperature
+        2) adjust radio station
+        3) monitor time
 
 Sub Object 4: Infotainment System
     Name 3 things you'll find on the:    Infotainment System
-        1)
-        2)
-        3)
+        1) radio display
+        2) GPS
+        3) temperature display
     Name 3 things you can do with the:   Infotainment System
-        1)
-        2)
-        3)
+        1) monitor radio
+        2) monitor temperature
+        3) monitor geo location
 
 Sub Object 5: Seat 
     Name 3 things you'll find on the:    Seat
-        1)
-        2)
-        3)
+        1) headrest
+        2) accelerate pedal
+        3) breaks pedal
```

Comment:
The headrest is a part of the seat, yes
the others, no.  They are not part of the seat.  they are a part of the Driver Controls.